docs/releasing.md

```diff
@@ -0,0 +1,74 @@
+## How to package and release Kinto.js
+
+In order to prepare and publish a new release, the following steps are required.
+
+### Start a release branch
+
+    $ git co -b prepare-1.1.0
+
+### Update the changelog
+
+We use [github-changelog-generator](https://github.com/skywinder/github-changelog-generator):
+
+    $ github_changelog_generator
+
+In the generated `CHANGELOG.md`, replace `HEAD` by `vX.Y.Z` in full changelog and link to the upcoming tag instead of `unreleased`.
+
+### Update in `package.json`
+
+* Bump the `version`;
+* Update the list of contributors if needed (`git shortlog -sne` gives a list of project contributors from the commit history);
+* Open a pull-request with the changes;
+* Wait for the tests to pass on [TravisCI](http://travis-ci.org/Kinto/kinto.js) before merging.
+
+### Merge the release branch, and push the tag
+
+    $ git checkout master
+    $ git rebase prepare-X.Y.Z
```

Comment:
forget it, both methods lead to the same result if fast-foward is possible. Probably matter of habits